FeedStoreChallenge/Helpers/ManagedCache+CoreDataClass.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ManagedCache+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Romeo Flauta on 5/15/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+public class ManagedCache: NSManagedObject {}
```

Comment:
`ManagedCache` is an implementation detail of `CoreDataFeedStore` thus it should not be public.
Also it is not inherited thus it should be final.

Please remove `public` modifier and make the class `final`.

```suggestion
final class ManagedCache: NSManagedObject {}
```